crates/welcome/src/base_keymap_setting.rs

```diff
@@ -25,17 +26,19 @@ impl Display for BaseKeymap {
             BaseKeymap::SublimeText => write!(f, "Sublime Text"),
             BaseKeymap::Atom => write!(f, "Atom"),
             BaseKeymap::TextMate => write!(f, "TextMate"),
+            BaseKeymap::None => write!(f, "None"),
         }
     }
 }
 
 impl BaseKeymap {
-    pub const OPTIONS: [(&'static str, Self); 5] = [
+    pub const OPTIONS: [(&'static str, Self); 6] = [
         ("VSCode (Default)", Self::VSCode),
         ("Atom", Self::Atom),
         ("JetBrains", Self::JetBrains),
         ("Sublime Text", Self::SublimeText),
         ("TextMate", Self::TextMate),
+        ("None (not recommended)", Self::None),
```

Comment:
We shouldn't show this option in the UI on the welcome screen as selecting it will make zed unusable and we don't want new users to get confused.